### PR TITLE
Add text formatting util functions

### PR DESCRIPTION
**Pull request by skazi0**
Ruby logger has some functions to get bold and colored outputs.
Regular wrapper functions were added to mimic the original output.

### CODE REVIEW

File: connect/utils.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package connect
```
Comment (djoreilly) [inline connect/utils.go:1]: Can we put these in main.go instead? We already have some logging stuff there. That way we avoid clogging the directory with too many files.

File: connect/utils.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+import "fmt"
+
+func logGreen(text string) string {
```
Comment (djoreilly) [inline connect/utils.go:5]: not sure about the "log" prefix, as technically it does not log, but just converts the text color. How about toGreen(), toRed() and toBold() ?

**Review by djoreilly — CHANGES_REQUESTED** [review]

File: connect/utils.go (hunk 2 above)
Comment (skazi0) [reply connect/utils.go]: I wanted to keep the naming from original code but these are not very important. I renamed to `greenText()` and `redText()`.

File: connect/utils.go (hunk 1 above)
Comment (skazi0) [reply connect/utils.go]: moved

**Comment by djoreilly** [comment]
From where will these be called? I think all the screen output we are doing is in cmd/suseconnect.go?  which is another package, which means we could have to call like connect.greenText() which is a bit unwieldy. Could just add these one-liners to the bottom of cmd/suseconnect.go instead?

**Comment by djoreilly** [comment]
oh - I see them in client.rb and not cli.rb. This lgtm then

**Review by djoreilly — APPROVED** [review]

**Pull request merged**